src/test/java/org/junitpioneer/jupiter/resource/ResourcesTests.java

```diff
@@ -43,9 +37,9 @@ class WhenNewResourceFactoryAppliedToParameterTests {
 		@DisplayName("then ::create is called")
 		@Test
 		void thenCreateIsCalled() {
-			ExecutionResults executionResults = PioneerTestKit
-					.executeTestClass(CountingResourceFactory1TestCases.class);
-			assertThat(executionResults.testEvents().debug().succeeded().count()).isEqualTo(1);
+			var results = executeTestClass(CountingResourceFactory1TestCases.class);
```

Comment:
When it comes to the use of `var`, I like Snyk's [cheat sheet](https://snyk.io/de/blog/local-type-inference-java-cheat-sheet/) (written by Simon Maple and Stuart Marks).

In the given case(s), I'd either do:

```java
var executionResults = /* ... */
```

Or:

```java
ExecutionResults results = /* ... */
```